### PR TITLE
Enable sec grpc

### PR DESCRIPTION
**Pull request by Eabramov84**

### CODE REVIEW

File: recipes-aos/aos-servicemanager/files/aos_servicemanager.cfg [hunk 1]
```diff
@@ -27,5 +33,6 @@
     "migration": {
         "migrationPath" : "/usr/share/servicemanager/migration",
         "mergedMigrationPath" : "/var/aos/servicemanager/mergedMigrationPath"
-    }
+    },
+    "enableDBusServer": false
```
Comment (al1img) [inline recipes-aos/aos-servicemanager/files/aos_servicemanager.cfg:37]: DBUS server should be enabled on VM.

File: recipes-aos/aos-servicemanager/files/aos_servicemanager.cfg [hunk 2]
```diff
@@ -3,13 +3,13 @@
         "CACert": "/etc/ssl/certs/rootCA.pem"
     },
     "serviceDiscovery": "https://aoscloud.io:9000",
-    "iamServer": "localhost:8090",
+    "iamServer": "aosiam:8090",
```
Comment (al1img) [inline recipes-aos/aos-servicemanager/files/aos_servicemanager.cfg]: This and following changes should be in `[sm] Update sm config to match latest changes ` and `[um] Update um config to match latest changes` accordingly.

File: recipes-aos/aos-servicemanager/aos-servicemanager_git.bbappend [hunk 3]
```diff
@@ -73,8 +73,13 @@ pkg_postinst_${PN}() {
         echo '127.0.0.1	wwwivi' >> $D${sysconfdir}/hosts
```
Comment (al1img) [inline recipes-aos/aos-servicemanager/aos-servicemanager_git.bbappend:73]: Fix commit message: it is SM recipe changes, so tag should be [sm].

File: recipes-aos/aos-servicemanager/files/aos_servicemanager.cfg (hunk 2 above)
Comment (Eabramov84) [reply recipes-aos/aos-servicemanager/files/aos_servicemanager.cfg]: fixed

File: recipes-aos/aos-servicemanager/aos-servicemanager_git.bbappend (hunk 3 above)
Comment (Eabramov84) [reply recipes-aos/aos-servicemanager/aos-servicemanager_git.bbappend:73]: fixed

File: recipes-aos/aos-servicemanager/files/aos_servicemanager.cfg (hunk 1 above)
Comment (Eabramov84) [reply recipes-aos/aos-servicemanager/files/aos_servicemanager.cfg:37]: added commnet to commit

**Review by al1img — APPROVED** [review]

**Pull request merged**